mmv1/products/netapp/volume.yaml

```diff
@@ -414,3 +416,17 @@ properties:
 #       description: |-
 #         Full name of the snapshot resource. Format: `projects/{{project}}/locations/{{location}}/volumes/{{volume}}/snapshots/{{snapshot}}`.
 #       required: true
+virtual_fields:
+  - !ruby/object:Api::Type::Enum
+    name: 'deletion_policy'
+    description: |
+      Policy to determine if the volume should be deleted forcefully.
+      Volumes may have nested snapshot resources. Deleting such a volume will fail.
+      Setting this parameter to FORCE will delete volumes including nested snapshots.
+    values:
+      - :DEFAULT
+      - :FORCE
+    default_value: :DEFAULT
+    ignore_read: true
```

Comment:
```suggestion
```
I don't think it's needed since it's a virtual field

Reply:
I honestly don't think `ignore_read` has effect on `virtual_fields`. The code that generates the downstream resource file from the yaml file can be found at https://github.com/GoogleCloudPlatform/magic-modules/blob/02700f26d946578bea2380669cb58583d148c3c6/mmv1/templates/terraform/resource.erb if it helps. Removing the `ignore_read` and re-generating the provider, you'll likely see no changes of the resource.

The above error happened earlier is likely fixed by adding `ignore_read_extra: "deletion_policy"` which adds the virtual field to `ImportStateVerifyIgnore` so it will be ignored during import test step

Reply:
Cool. Can you update the change in this PR as well? I think we'd want to separate the `deletion_policy` implementation from the CRR and ship this one first when it's ready. Thanks!

Reply:
oh gosh, I thought I had updated this one a day ago. Will do it immediately. As soon as this one is merged, I will remove it from the CRR one. I want to keep it in meanwhile to make the test run fine.